dowhy/causal_graph.py

```diff
@@ -33,6 +33,11 @@ def __init__(self,
         mediator_names = parse_state(mediator_names)
         self.logger = logging.getLogger(__name__)
 
+        
+        if isinstance(graph, str):
+            graph=graph.replace("\n", " ")
+
+
```

Comment:
a good practice is to avoid any unnecessary lines. you can remove the extra new lines 36 and 39.